src/config/prod.ts

```diff
@@ -0,0 +1,7 @@
+const config = {
+	secrets: {
+		jwt: 'jwtprod',
+	},
```

Comment:
Could be empty since we already have common secrets.jwt = process.env.JWT_SECRET